dug_6_osfv.md

```diff
@@ -0,0 +1,152 @@
+class: center, middle, intro
+
+# &#x1F386; Dasharo Open Source Firmware Validation &#x1F386;
+
+.center[<img src="/remark-templates/dasharo-presentation-template/images/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px">]
+
+???
+
+---
+
+# TBD: Agenda
+
+* Introduction to Dasharo Open Source Firmware Validation (OSFV)
```

Comment:
I would use intro from last presentation and try to improve it, so it better reflects current state of the art or maybe even little bit show vision where we going with usage of OSFV and its interaction with lab infrstructure.

Reply:
Let's try to present various users perspective:
- QEMU validation of DTS
- dev use cases - usage while adding regression tests, test driven bug fixing, usage while developing, code rebase verification
- val use cases - regression testing, release testing